CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ cmake_minimum_required(VERSION 3.2)
 # There is no C per se in WDT but if you use CXX only here many checks fail
 # Version is Major.Minor.YYMMDDX for up to 10 releases per day (X from 0 to 9)
 # Minor currently is also the protocol version - has to match with Protocol.cpp
-project("WDT" LANGUAGES C CXX VERSION 1.29.1703251)
+project("WDT" LANGUAGES C CXX VERSION 1.30.1703270)
```

Comment:
You will probably have to change the version to 31, as I am using 30 for an encryption fix.